.env.example

```diff
@@ -0,0 +1,9 @@
+export ACCESS_KEY=123456
+export SENTRY_DSN=
+
+export SPREADSHEETS_MICROSERVICE_URL=
+export SPREADSHEETS_MICROSERVICE_API_KEY=
+export SPREADSHEET_KEY=
+export WORKSHEET_TITLE=
```

Comment:
Looks like this is safe to remove with commit ed566a7f93c6bc38c040b201f3b786ac2e817332

Reply:
done!  thanks!